sys-apps/shadow/shadow-4.4.ebuild

```diff
@@ -117,15 +131,14 @@ src_install() {
 	newins etc/login.defs login.defs
 	dosym ../usr/share/shadow/login.defs /etc/login.defs
 
+	set_login_opt CREATE_HOME yes
```

Comment:
I dug into it a bit and it looks like right now it's only used in `useradd` for defaulting `-m` vs `-M`.

There is a line in "TODO" to use it for "newusers", but no code related to that.